internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/restart/TestRestartServerAfterDeletingSstFilesWithActiveActiveIngestion.java

```diff
@@ -347,6 +361,40 @@ public void testActiveActiveStoreWithRMDAndRestartServer(boolean deleteSSTFiles,
           ExecutionStatus.COMPLETED);
     });
 
+    // create a kafka consumer to check for the messages with leaderCompletedStatus
+    Properties properties = new Properties();
+    properties.setProperty(ConfigKeys.KAFKA_BOOTSTRAP_SERVERS, pubSubBrokerWrapper.getAddress());
+    ApacheKafkaConsumerConfig apacheKafkaConsumerConfig =
+        new ApacheKafkaConsumerConfig(new VeniceProperties(properties), "testConsumer");
+    ApacheKafkaConsumerAdapter consumer =
```

Comment:
Let's not tie up the test to any specific pubsub client. Could you please update it to use `PubSubConsumerAdapter`? See this example here: https://github.com/linkedin/venice/blob/d07f18606d10d9aaf9802ed7ed291767c2e64080/internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/pubsub/api/consumer/PubSubConsumerAdapterTest.java#L117

Reply:
Thanks for the pointer, changed it.